modules/bindgen/src/main/scala/render/enumeration.scala

```diff
@@ -38,17 +45,21 @@ def enumeration(model: Def.Enum, line: Appender)(using
 
       line(lhs + " = " + rhs)
     }
-    line(s"inline def getName(inline value: $opaqueType): Option[String] =")
+    line(
+      s"inline def getName(inline value: $opaqueType): Option[String] = $openDefDelimiter"
+    )
     nest {
```

Comment:
In rendition I have this [`block`](https://github.com/indoorvivants/rendition/blob/main/modules/core/src/main/scala/Rendering.scala#L141-L160) method that handles open/close delimiter.

Perhaps it's possible to define it somewhere in the render package, and define a couple of specific variants that are chosen locally based on the config.

This way it can dramatically reduce the number of `line + nest + appendCloseDelimiter` separate calls, which are error prone